draft-ietf-rats-corim.md

```diff
@@ -641,6 +641,9 @@ The `triples-map` contains all the CoMID triples broken down per category.  Not
 all category need to be present but at least one category MUST be present and
 contain at least one entry.
 
+The supply chain entity that is responsible for providing a triple (i.e., Reference Values or Endorsed Values) is by default the CoRIM signer.
+The signer of a triple is said to be its *authority*.
+
```

Comment:
This section is about the triples map generally. It seems premature to go into detail about authorized-by here. In about 100 lines further we introduce `authorized-by` and talk about it in detail. Wouldn't this text be better suited for that section?

Reply:
ok, then please move my suggested text to the section, which is 100 lines further from here (line 646).

But then please modify line 646 as follows.

` In most cases, the supply chain entity that is responsible for providing a triple (i.e., Reference Values or Endorsed Values) is by default the CoRIM signer.
The signer of a triple is said to be its *authority*` However in special cases, more authorities may be involved in triples other than the CoRIM Signer. Please refer section XYZ for details`

Reply:
Note that issue #293 discusses use cases for authorized-by. There was consensus for use case 1 (authorized-by in conditions used for searching ECTs) makes sense.
There wasn't consensus for any of the other use cases, all of which had some form of including authorized-by in an addition field.
The suggested text seems to anticipate one of the use cases other than use case 1?

Reply:
XYZ should be the place where we define triple semantics and each element of the triple.!

Reply:
Section 5.1.4 Triples?

Reply:
I think, we need to be clear if we are ONLY catering to Use Case 1, then we need to add a github issue, in the document, to state that their may be cases, where more than one authorities may be involved in supplying measurements to a Verifier. Such scenarios are not explored yet. However the place holder is present in CDDL to express authority of supplier of multiple measurement Claims(via measurement-map) and Triples via expression of multiple triples in a CoRIM.

Reply:
> Section 5.1.4 Triples?

Yes

Reply:
Sticking to use case 1. That means only the condition ECTs will populate authorized-by. It might still be reasonable for search criteria to differ given multiple measurement-maps. For example, COSE_Sign1 allows multiple signers. That would be realized by ECT.authority having multiple entries. It is reasonable for the condition in one authorized-by to name one of the signers and for another authorized-by to name another. 

The current schema isn't broken, but we don't necessarily provide guidance text for all the possible ways the schema might be used. That might be better suited for an informational RFC or a white paper of some kind.

Reply:
> Sticking to use case 1. That means only the condition ECTs will populate authorized-by. It might still be reasonable for search criteria to differ given multiple measurement-maps. For example, COSE_Sign1 allows multiple signers. That would be realized by ECT.authority having multiple entries. It is reasonable for the condition in one authorized-by to name one of the signers and for another authorized-by to name another.
> 
> The current schema isn't broken, but we don't necessarily provide guidance text for all the possible ways the schema might be used. That might be better suited for an informational RFC or a white paper of some kind.

The CDDL and explanation is out of sync, all I am trying to do is to clarify what is in the document and what CDDL is representing, just trying to let them be in sync.

Reply:
Perhaps a compromise for wording can be 

If additions populate authorized-by with any key different from the CoRIM Issuer's, then the CoRIM Issuer MUST be an authorized delegate of the alternate key for the purpose of assigning authority to the Claims in question.
By default, no key is an authorized delegate for any other key for any Claim.
The definition of an authorized delegate is a policy decision outside the scope of this document.

Reply:
> If additions populate authorized-by

While I agree with the guidance text above, I can't point to any transformation text that populates additions. This was by design because we said we'd only focus on use case 1 for now.

Reply:
> The CDDL and explanation is out of sync, all I am trying to do is to clarify what is in the document and what CDDL is representing, just trying to let them be in sync.

I'm not sure what is out of sync (if my latest text didn't clarify it sufficiently). These edits are in the Triples section already, so there's no point in self-referencing it. 

I honestly think the original text was sufficient for this section. It merely introduced some terminology that is used later. The new text seems reasonable too, but is on the edge of getting ahead of ourselves as we're just explaining what a triple is right now.